core/src/Kokkos_View.hpp

```diff
@@ -494,8 +494,18 @@ constexpr bool is_assignable(const Kokkos::View<ViewTDst...>& dst,
 
 namespace Kokkos {
 
+// FIXME_OPENMPTARGET - The `declare target` is needed for the Intel GPUs with
+// the OpenMPTarget backend
+#if defined(KOKKOS_ENABLE_OPENMPTARGET) && defined(KOKKOS_ARCH_INTEL_PVC)
+#pragma omp declare target
+#endif
```

Comment:
Similarly, here
```suggestion
#if defined(KOKKOS_ENABLE_OPENMPTARGET) && defined(KOKKOS_COMPILER_INTEL)
#pragma omp declare target
#endif
```

Reply:
Updated to make it generic to all Intel GPUs with OpenMPTarget backend.